
package publicadores;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.4.4
 * 2021-11-02T02:11:11.804-03:00
 * Generated source version: 3.4.4
 */

@WebFault(name = "UsuarioExisteException", targetNamespace = "http://publicadores/")
public class UsuarioExisteException_Exception extends java.lang.Exception {

    private publicadores.UsuarioExisteException faultInfo;

    public UsuarioExisteException_Exception() {
        super();
    }

    public UsuarioExisteException_Exception(String message) {
        super(message);
    }

    public UsuarioExisteException_Exception(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public UsuarioExisteException_Exception(String message, publicadores.UsuarioExisteException usuarioExisteException) {
        super(message);
        this.faultInfo = usuarioExisteException;
    }

    public UsuarioExisteException_Exception(String message, publicadores.UsuarioExisteException usuarioExisteException, java.lang.Throwable cause) {
        super(message, cause);
        this.faultInfo = usuarioExisteException;
    }

    public publicadores.UsuarioExisteException getFaultInfo() {
        return this.faultInfo;
    }
}
